src/Configurator/ComposerScriptsConfigurator.php

```diff
@@ -49,7 +50,7 @@ public function unconfigure(Recipe $recipe, $scripts, Lock $lock)
     public function update(RecipeUpdate $recipeUpdate, array $originalConfig, array $newConfig): void
     {
         $json = new JsonFile(Factory::getComposerFile());
-        $jsonPath = ltrim(str_replace($recipeUpdate->getRootDir(), '', $json->getPath()), '/\\');
+        $jsonPath = Path::makeRelative($json->getPath(), $recipeUpdate->getRootDir());
```

Comment:
Flex has no dependencies other than `composer-plugin-api`, so it is not possible to use an external package.